release/pkg/file_reader.go

```diff
@@ -349,3 +327,29 @@ func ReadHttpFile(uri string) ([]byte, error) {
 
 	return data, nil
 }
+
+func CheckFilesEquals(t *testing.T, actualPath, expectedPath string) {
```

Comment:
should this method be in a test package?

Reply:
Done